app/helpers/remote_execution_helper.rb

```diff
@@ -194,4 +193,13 @@ def time_in_words_span(time)
     end
   end
 
+  def documentation_button_rex(section = '')
+    url = "http://theforeman.org/plugins/foreman_remote_execution/\
+#{ForemanRemoteExecution::VERSION}/index.html#" + section
```

Comment:
Wouldn't we need something like `ForemanRemoteExecution::VERSION.split('.').take(2).join('.')` since this resolves to 0.1.2?